### ISSUE TITLE
Missing Nan::SetCallAsFunctionHandler?

### ISSUE THREAD

**Issue by unbornchikken** [nodejs/nan#427]
Hi,

I have an ObjectTemplate and trying to set its function handler via SetCallAsFunctionHandler:

```js
tmpl->SetCallAsFunctionHandler(MyClass::New);
```

MyClass::New is just a regular NAN_METHOD. But the above doesn't compile of course:

```
error: invalid conversion from ‘Nan::NAN_METHOD_RETURN_TYPE (*)(Nan::NAN_METHOD_ARGS_TYPE) {aka void (*)(const Nan::FunctionCallbackInfo<v8::Value>&)}’ to ‘v8::FunctionCallback {aka void (*)(const v8::FunctionCallbackInfo<v8::Value>&)}’ [-fpermissive]

```

There should be Nan::SetCallAsFunctionHandler() for that puropse, right? I cannot find it. If it's missing could someone please suggest me a quick workaround?

**Comment by kkoopa** [comment]
Yes, it seems to be missing. There is no quick workaround, short of actually implementing the missing function.

**Comment by unbornchikken** [comment]
Do you have plan to support it in a next release? There is a +1 for that.

**Comment by unbornchikken** [comment]
I can see some progress on this, but it's really hard to follow from there. Does it landed in a release yet?

**Comment by kkoopa** [comment]
Not yet. 2.1
